src/runtime/disco/loader.cc

```diff
@@ -60,6 +61,10 @@ class ShardLoaderObj : public Object {
     const FileRecord* file;
     const ParamRecord* param;
     int shard_dim;
+    PackedFunc f_reorder_fp16_;
```

Comment:
I'm thinking of future cases where we might need more kinds of preprocessing other than reorder. If we let users pass a preprocess function's base name in shard info, like r`eorder_qkv`, and look for corresponding dtype variant `reorder_qkv_fp16` in IRModule automatically, it might be more flexible. If we want to do that, we can look up preprocess func when we call `Shard` instead of `Create`